src/main/java/com/volmit/iris/util/json/JSONCleaner.java

```diff
@@ -0,0 +1,66 @@
+package com.volmit.iris.util.json;
```

Comment:
This should go where its used, in the studio commands

Reply:
Question for this one, should I remove `#beautify` in favour of `#clean`?

Reply:
If you saw my clean command, it allowed you to add params like beautify rewriteObjects & update workspace all in the clean command. So yes. Idk what happened to my command but you could use that one since it uses a multicore multi-task job executor for it, looks super sexy ingame.

Reply:
OK
Ten